mlflow/store/artifact/databricks_artifact_repo_resources.py

```diff
@@ -41,6 +60,18 @@ def __init__(self, id_: str, call_endpoint: Callable[..., Any]):
         self.id = id_
         self.call_endpoint = call_endpoint
         self.artifact_root = self.get_artifact_root()
+        self.relative_path = self.get_rel_path
+
+    def get_relative_path(self) -> str:
+        # Fetch the artifact root for the MLflow resource associated with `artifact_uri` and compute
+        # the path of `artifact_uri` relative to the MLflow Run's artifact root
```

Comment:
```suggestion
        # the path of `artifact_uri` relative to the MLflow resource's artifact root
```

Do we need to update any similar inline comments?